model/train.py

```diff
@@ -415,6 +420,8 @@ def decode_and_calculate_acc(*, p_pred_step, p_init_cache, target, config,
     predicted = p_pred_step(pred_batch['inputs'], target, cache, decode.EOS_ID,
                             max_predict_length)
     predicted_list = [tohost(predicted)]
+    if use_annotations == True:
```

Comment:
If `use_annotations` is expect to be boolean, then this would be simpler:
```suggestion
    if use_annotations:
```